test/functional/sql-tag/sql-tag-parameters-better-sqlite3.ts

```diff
@@ -0,0 +1,212 @@
+                ])
+
+                const minValue = 15
+                const maxValue = 25
+                const namePattern = "b"
+                const [example] = await connection.sql`SELECT * FROM example
+                    WHERE value > ${minValue}
+                    AND value < ${maxValue}
+                    AND name LIKE ${namePattern}
+                    ORDER BY value DESC`
+
+                expect(example.id).to.equal("test2")
+                expect(example.value).to.equal(20)
+            }),
+        ))
+
+    it("should handle SQL tag parameters with NULL values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(Example)
+
+                await repo.save([
+                    { id: "null1", value: null },
+                    { id: "null2", value: 10 },
+                ])
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE value IS ${null}
+                `
+
+                const ids = examples.map((e: Example) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["null1"])
+            }),
+        ))
+
+    it("should handle SQL tag parameters with boolean values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(Example)
+
+                await repo.save([
+                    { id: "true1", active: true },
+                    { id: "false1", active: false },
+                ])
+
+                const value = 1
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE active = ${value}
+                `
+
+                const ids = examples.map((e: Example) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["true1"])
+            }),
+        ))
+
+    it("should handle SQL tag parameters with date values", () =>
+        Promise.all(
+            connections.map(async (connection) => {
+                const repo = connection.getRepository(Example)
+                const now = new Date()
+                const yesterday = new Date(now.getTime() - 24 * 60 * 60 * 1000)
+                const nowISO = now.toISOString()
+                const yesterdayISO = yesterday.toISOString()
+
+                await repo.save([
+                    { id: "today", createdAt: nowISO },
+                    { id: "yesterday", createdAt: yesterdayISO },
+                ])
+
+                const examples = await connection.sql`
+                    SELECT * FROM example WHERE "createdAt" > ${yesterdayISO}
+                `
+
+                const ids = examples.map((e: Example) => e.id)
+
+                expect(examples).to.have.length(1)
+                expect(ids).to.have.members(["today"])
+            }),
+        ))
+
```

Comment:
I think we can drop this test, since we don't expect the `sql` tag to do anything special with dates - and this test is binding a string to the parameter anyway.

```suggestion
```